src/database/mod.rs

```diff
@@ -150,19 +150,25 @@ impl Database {
     /// let db = Database::open_with_mode("/path/to/db", DatabaseOpenMode::default()).unwrap();
     /// // work with the database
     /// ```
-    pub fn open_with_mode<P: Into<Vec<u8>>>(path: P, open_mode: DatabaseOpenMode) -> Result<Database> {
+    pub fn open_with_mode<P: Into<Vec<u8>>>(
+        path: P,
+        open_mode: DatabaseOpenMode,
+    ) -> Result<Database> {
```

Comment:
Just for the information, why this got reformatted?

Reply:
The line was too long (more than 80 symbols), so `rustfmt` broke it down.

Reply:
I see. I guess I'll need to tweak rustfmt settings since I always set the line length limit to 100+. Not important for this PR though.